lib/amazonka/BUILD.bazel

```diff
@@ -12,12 +12,14 @@ haskell_library(
         "src/Amazonka/Auth/Exception.hs",
         "src/Amazonka/Auth/InstanceProfile.hs",
         "src/Amazonka/Auth/Keys.hs",
+        "src/Amazonka/Auth/SSO.hs",
```

Comment:
Oops, did I forget some Bazel-related bookkeeping when doing the SSO stuff?

Reply:
Yeah, and I didn't notice either until I re-ran `scripts/update-gazelle`, which generates `BUILD.bazel` from cabal files.